OpenRA.Mods.Common/Widgets/Logic/Ingame/IngamePowerBarLogic.cs

```diff
@@ -26,9 +26,10 @@ public IngamePowerBarLogic(Widget widget, ModData modData, World world)
 			var powerManager = world.LocalPlayer.PlayerActor.Trait<PowerManager>();
 			var powerBar = widget.Get<ResourceBarWidget>("POWERBAR");
 
+			var powerUsage = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
 			powerBar.GetProvided = () => powerManager.PowerProvided;
 			powerBar.GetUsed = () => powerManager.PowerDrained;
-			powerBar.TooltipFormat = modData.Translation.GetString(PowerUsage) + ": {0}/{1}";
+			powerBar.TooltipFormat = powerUsage;
```

Comment:
`TooltipFormat` is just a string, not a function, so setting this here once is ok.

Reply:
you're right! I tackled the wrong translation, the issue was in IngamePowerCounterLogic